_haikus/The_Presence.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: The Presence
+author: Hanike
+---
+
+There is a presence:<br>
+Silently, the cat watches you<br>
+From maddening darkness...<br>
```

Comment:
This final line also needs 1 fewer syllable

Reply:
_Noooo~_ <(ToT)>